app/models/WrongWithMovement.scala

```diff
@@ -66,6 +67,21 @@ object WrongWithMovement extends Enumerable.Implicits {
     }
   }
 
+  def itemShortageOrExcessOptions(implicit messages: Messages) = {
+    Seq(
+      RadioItem(
+        content = Text(messages(s"itemShortageOrExcess.shortageOrExcess.$Less")),
+        value = Some(Less.toString),
+        id = Some(Less.toString)
+      ),
```

Comment:
is it worth matching the IDs on the page and database entries to be shortage and excess instead of less and more?